MTM-Frontend/src/pages/HomeLayout/index.tsx

```diff
@@ -11,6 +11,8 @@ import { TopBar } from "../../components/NavigationBar/TopBar";
 export const HomeLayout: React.FC = () => {
   const [fact, setFact] = useState<string>("");
   const { currentUser } = useAuth();
+  const location = useLocation();
+  const isHomeIndex = location.pathname === "/home";
 
   useEffect(() => {
```

Comment:
Let's get rid of this since this is just an example

Reply:
what do you mean? like the unnecessary method imports?

Reply:
The useEffect hook to grab the example data

Reply:
Done